packages/utils/build-tools/src/configureMetro.ts

```diff
@@ -7,6 +7,7 @@ import { getRNVersion, getAllPlatforms, getAllReactNativePaths, PlatformValue, f
 import { getPackageInfo, findGitRoot } from 'just-repo-utils';
 import blacklist from 'metro-config-60/src/defaults/blacklist';
 import { mergeConfigs } from './mergeConfigs';
+import { getDefaultConfig } from 'metro-config-60';
```

Comment:
I need to find the right way to make this version agnostic.  It may need a plugin module similar to haul for 0.60, 0.61, etc.  (obviously not a comment on this change)